lib/Backends/CPU/LLVMIRGen.cpp

```diff
@@ -1459,10 +1459,15 @@ void LLVMIRGen::generateLLVMIRForInstr(llvm::IRBuilder<> &builder,
     auto *kernel = emitConstSizeT(builder, CI->getKernel());
     auto *stride = emitConstSizeT(builder, CI->getStride());
     auto *pad = emitConstSizeT(builder, CI->getPad());
+    auto *group = emitConstSizeT(builder, CI->getGroup());
 
     size_t inChannels = src->dims()[3];
     size_t outChannels = src->dims()[3];
 
+    assert(inChannels % CI->getGroup() == 0 &&
```

Comment:
this is probably redundant, can you add an instruction verifier for that instead, similar to CPUMaxSplat.